dev/server.ts

```diff
@@ -0,0 +1,159 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+
+import * as babel from '@babel/core';
+import * as chokidar from 'chokidar';
+import * as express from 'express';
+import * as morgan from 'morgan';
+import * as portfinder from 'portfinder';
+
+import { makeListing } from '../src/common/tools/crawl.js';
+
+// Make sure that makeListing doesn't cache imported spec files.
+process.env.STANDALONE_DEV_SERVER = '1';
+
+// Import the project's babel.config.js. We'll use the same config for the runtime compiler.
+const babelConfig = {
+  ...require(path.resolve(__dirname, '../babel.config.js'))({
+    cache: () => {
+      /* not used */
+    },
+  }),
+  sourceMaps: 'inline',
+};
+
+// Caches for the generated listing file and compiled TS sources to speed up reloads.
```

Comment:
Comment what the key is here (request path?)

Reply:
Done.